test/unit/api/cpp/solver_black.cpp

```diff
@@ -498,6 +512,22 @@ TEST_F(TestApiBlackSolver, mkFiniteFieldElem)
             d_solver.mkFiniteFieldElem("6", f));
   ASSERT_EQ(d_solver.mkFiniteFieldElem("1", f),
             d_solver.mkFiniteFieldElem("8", f));
+
+  
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldElem("0", f, 2));
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldElem("101", f, 3));
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldElem("-10", f, 7));
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldElem("abcde", f, 16));
+
+  ASSERT_EQ(d_solver.mkFiniteFieldElem("0", f, 2),
+            d_solver.mkFiniteFieldElem("0", f, 3));
+  ASSERT_EQ(d_solver.mkFiniteFieldElem("11", f, 2),
+            d_solver.mkFiniteFieldElem("10", f, 3));
+  ASSERT_EQ(d_solver.mkFiniteFieldElem("1010", f, 2),
+            d_solver.mkFiniteFieldElem("A", f, 16));
+
+  ASSERT_EQ(d_solver.mkFiniteFieldElem("-22", f, 3),
+            d_solver.mkFiniteFieldElem("10", f, 6));
```

Comment:
These tests should be added to a `test_mk_finite_field_elem` test in `test/unit/api/python/test_solver.py`. It seems that this test does not exist (@alex-ozdemir looking at you :eyes: ). @Sarkoxed would you mind adding these plus the ones that already existed here to such a new test?